tests/test_fork_point.py

```diff
@@ -34,6 +34,10 @@ def test_fork_point(self) -> None:
                 .new_branch("feature")
                 .commit('feature commit.')
         )
+        # Clean-up environment variables
+        os.environ.pop('GIT_COMMITTER_DATE', None)
```

Comment:
Nit: in the unlikely circumstance of the above statement failing, these 2 vars will NOT be popped... and thus will likely be propagated to other test cases. If so, then pls use `try..finally` here

Reply:
@PawelLipski  Hmm, i especially chose `os.environ.pop('GIT_COMMITTER_DATE', None)` so in the unlikely scenario that these variables don't get set in the previous steps, the test won't fail. I don't see any other reason for these statements to fail, wdyt?

Reply:
@PawelLipski  Hmm, i especially chose `os.environ.pop('GIT_COMMITTER_DATE', None)` so in the unlikely scenario that these variables don't get set in the previous steps, the test won't fail. I don't see any other reason for these statements to fail, wdyt?

Reply:
No! I mean, what if `self ... commit(...)` fails? 🤔 `GIT_COMMITTER_DATE` will remain set!